docs/api/modal.md

```diff
@@ -74,7 +74,9 @@ Developers can set `canDismiss` to a boolean value. If `canDismiss` is `true`, t
 
 Setting a boolean value should be used when you need to require a particular action to be taken prior to a modal being dismissed. For example, if developers want to require that a "Terms of Use" checkbox is checked prior to closing the modal, they could set `canDismiss` to `false` initially and update it to `true` when the checkbox is checked.
 
-TODO: Playground Example
+import CanDismissBooleanExample from '@site/static/usage/modal/can-dismiss/boolean/index.md';
+
+<CanDismissBooleanExample />
```

Comment:
For the React and Vue Stackblitzes (and possibly Angular too), after closing the modal and re-opening it, the checkbox starts unchecked, but the modal can be closed: https://user-images.githubusercontent.com/90629384/166950991-18c6ed4f-3eeb-4a0a-84ac-9885b13543cd.mp4

Reply:
Good catch 🤔  for simplicity, I'm just going to persist the checked state of the `ion-checkbox`. So they can open the modal, check the checkbox and dismiss the modal. If they re-open the modal, the checkbox will already be pre-selected, since the state is persisted. This will visually make sure that that checked state and the dismiss button are always in sync.